<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-KB6UDKRL" failures="5" tests="10" name="TestRunner-Firefox" time="113.746" errors="0" timestamp="2021-10-18T10:22:08 IST">
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="5.009"/>
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="4.85"/>
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="4.884"/>
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="5.025"/>
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="5.623"/>
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="5.733">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="5.622">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="5.654">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="5.561">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="4.82">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
</testsuite> <!-- TestRunner-Firefox -->
